### PR TITLE
New utility file: TokenChecker

### PR DESCRIPTION
**Pull request by Chr1Z93**
New file added to `src\core\token`: TokenChecker
Just contains one function for now: `isChaosToken(obj)`
- returns true if the object is a chaos token and false if not
- implemented that for the token remover and playermat discard button (to return chaos tokens on cards like unrelenting)

Additionally:
- bless / curse token bag: added missing name to tokesn, changed contained object to be 10x the same (no need for multiple files)

### CODE REVIEW

File: src/util/isChaosToken.ttslua [hunk 1]
```diff
@@ -0,0 +1,31 @@
+do
```
Comment (Buhallin) [inline src/util/isChaosToken.ttslua:1]: I don't think we're going to want a pattern with one function per file.  That's going to explode fast.  Should probably make this something like ChaosTokenUtils and attach the function to that

File: objects/AllPlayerCards.15bb07/Unrelenting1.45386d.ttslua [hunk 2]
```diff
@@ -1,5 +1,3 @@
-ASSEMBLY_POSITION = {69.08, 4, 36.45}
```
Comment (Buhallin) [inline objects/AllPlayerCards.15bb07/Unrelenting1.45386d.ttslua:1]: Should there be more changes in this file?

**Review by Buhallin — CHANGES_REQUESTED** [review]

File: src/util/isChaosToken.ttslua (hunk 1 above)
Comment (Buhallin) [reply src/util/isChaosToken.ttslua:1]: Actually let's just make this TokenUtils or TokenChecker - we can reasonably expand it to isClue(), isDoom(), etc

File: objects/AllPlayerCards.15bb07/Unrelenting1.45386d.ttslua (hunk 2 above)
Comment (Chr1Z93) [reply objects/AllPlayerCards.15bb07/Unrelenting1.45386d.ttslua:1]: Nope, just a removal of a dead table (noticed this while looking at all the cards that can have chaos tokens on them)

File: src/util/isChaosToken.ttslua (hunk 1 above)
Comment (Chr1Z93) [reply src/util/isChaosToken.ttslua]: Done

**Review by Buhallin — APPROVED** [review]

**Pull request merged**